Susi/Helpers/ControllerConstants.swift

```diff
@@ -257,6 +257,11 @@ class ControllerConstants {
         static let roomsIcon = UIImage(named: "room-icon")
     }
     
+    struct BannerNotification {
+        static let dangerTitle =  "No Internet Connection! Connect to Internet"
+        static let successTitle = "Connected To Internet , Welcome Back"
```

Comment:
Change it to `Connected to the internet!`

Reply:
changes done , please review